detailed_docs/RC/resource_allocation.md

```diff
@@ -150,32 +187,37 @@ _Steps:_
 
 1. User disables RC functionality
 2. HMI sent notification to SDL that RC functionality is disabled
-3. SDL assigns HMILevel NONE to RC_app_1 and RC_app_2 and unsubscribes applications from module_1
+3. SDL assigns HMILevel NONE to RC_app_1 and RC_app_2 and unsubscribes applications from module_1  
```

Comment:
please see https://github.com/smartdevicelink/sdl_core/issues/2173 and https://github.com/smartdevicelink/sdl_evolution/issues/531, we require keep the current HMI level

Reply:
@yang1070, thank you. these changes will be added after accepting the proposal in the corresponding feature branch with a separate PR.